test/src/e2e.dynval/dv.n+1.test.ts

```diff
@@ -0,0 +1,249 @@
+// Copyright 2019 Kodebox, Inc.
+// This file is part of CodeChain.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import * as chai from "chai";
+import { expect } from "chai";
+import * as chaiAsPromised from "chai-as-promised";
+import * as stake from "codechain-stakeholder-sdk";
+import "mocha";
+
+import { validators as originalValidators } from "../../tendermint.dynval/constants";
+import { faucetAddress, faucetSecret } from "../helper/constants";
+import { PromiseExpect } from "../helper/promise";
+import { withNodes } from "./setup";
+import CodeChain from "../helper/spawn";
+
+chai.use(chaiAsPromised);
+
+const [betty, , , ...otherDynValidators] = originalValidators;
```

Comment:
Would you change this?
```ts
const otherDynValidators = originalValidators.slice(0, 7);
const betty = originalValidators[7];
```
It is more clear that there are 7 validators in `otherDynValidators`, and it is easier than counting empty commas.

Reply:
I introduced slicing